.github/workflows/tasks.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       Env: ${{ inputs.Environment }}
       Timeout: ${{ inputs.Timeout }}
       Region: ${{ inputs.Region }}
-      Command: 'hypothesis migrate current'
+      Command: 'alembic -c conf/alembic.ini current'
```

Comment:
We'll have to test this in QA and production to make sure it works.

This makes h's `tasks.yml` the same as in other apps, which'll make it easier to move `tasks.yml` into the cookiecutter in future.